### ISSUE TITLE
ReactNative examples?

### ISSUE THREAD

**Issue by delkant** [ConnectyCube/connectycube-js-samples#7]
Hi, 
Do you have plans for adding react native examples?

Thanks,

**Comment by DaveLombardo** [comment]
Hi @delkant 
yes, a Chat sample is in progress and will be available here in 2 weeks

Do you use RN+Angular or RN+Vue ?

**Comment by delkant** [comment]
Not so sure about the suggested combination RN + X.
I will need to integrate a chat to an existing App, I was reviewing alternatives when I end up here, I am looking for a pure RN library.

**Comment by DaveLombardo** [comment]
@delkant  sorry man, I messed up again the React Native and Native Script stuff  :) 
Too many 'Native' things in my life ;-DD 

Please ignore my previous message

**Comment by Seunope** [comment]
@DaveLombardo  Please where can i get  React Native example for ConnectyCube

**Comment by DaveLombardo** [comment]
@Seunope we have release a Native Script code sample for Chat

https://github.com/ConnectyCube/connectycube-js-samples/tree/master/sample-chat-nativescript-core

our next step - work on React Native

**Comment by Iris-germanica** [comment]
@Seunope  We have released React Native sample for file upload: https://github.com/ConnectyCube/connectycube-js-samples/tree/master/RNUploadFiles 

Here is a live demo: https://expo.io/@ccvlad/RNUploadFiles

Project contains the following features implemented:
- User login to ConnectyCube platform
- Possibility to make photo via device camera
- Possibility to pick images from device gallery
- Upload the chosen image file to ConnectyCube storage

More React Native samples are coming soon.
Your feedback is always welcome!

**Comment by shubham-2710** [comment]
Hi, Any plans for React Native Chat Example ?

**Comment by DaveLomber** [comment]
@shubham-2710 yes, will be released pretty soon
We will notify you here

**Comment by Iris-germanica** [comment]
@shubham-2710 We have released React Native Chat samples. Please check it here: https://developers.connectycube.com/js/code-samples?id=chat-code-sample
should you have any further questions, pleased let us know

**Comment by jpgarcia99** [comment]
When will you release a react-native videochat sample?

**Comment by jraleman** [comment]
I am also interested in the video call sample... Is it in development?

**Comment by Iris-germanica** [comment]
@jpgarcia99 , @jraleman 
Yes, we are working on video chat samples in React Native currently. 
We have already pushed a working branch with RN video example [here](https://github.com/ConnectyCube/connectycube-js-samples/tree/feature/rn-video/RNVideoChat) also beta version of SDK is there
There is still some work to be done which we are heavily working on right now

But you can already start integrating it in your app.

Basically the Video Chat API is pretty the same we have for [JS Web](https://developers.connectycube.com/js/videocalling)
The only one difference is video view rendering which you can whack in our sample already

We will keep you posted on future updates.

let us also know if there are any issues/questions so far

**Comment by DaveLomber** [comment]
@jpgarcia99  @jraleman 
A 1st beta version is available here https://github.com/ConnectyCube/connectycube-js-samples/tree/feature/rn-video/RNVideoChat 

please check

we continue work on it to make it more advance, so new commits will come soon

**Comment by jraleman** [comment]
Thank you! I really appreciate the work that you guys have put into this project!

**Comment by DaveLomber** [comment]
both Chat & Video chat samples are available here 
https://developers.connectycube.com/reactnative/code-samples

so closing the issue
